### PR TITLE
fix(atomic): fixed atomic-result-image size on chrome and safari 15

### PR DESCRIPTION
**Pull request by btaillon**
https://coveord.atlassian.net/browse/KIT-1200

`aspect-ratio` doesn't work on Safari 14, we'd have to use Javascript. Should we support it?

### CODE REVIEW

File: packages/atomic/src/components.d.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export namespace Components {
          */
         "numberOfValues": number;
         /**
-          * The sort criterion to apply to the returned facet values. Possible values are 'score', 'alphanumeric', 'occurrences', and 'automatic'. TODO: add automatic (occurences when not expanded, alphanumeric when expanded)
```
Comment (btaillon) [inline packages/atomic/src/components.d.ts:61]: Unrelated to this PR.

**Review by samisayegh — APPROVED** [review]
> Should we support it?

If there are alternative CSS rules that could give the same result, I'd lean towards that, especially if supporting the property means adding polyfills.

**Comment by ThibodeauJF** [comment]
Yes, v14 is quite recent 😢

**Review by ThibodeauJF — APPROVED** [review]

**Comment by btaillon** [comment]
> If there are alternative CSS rules that could give the same result, I'd lean towards that, especially if supporting the property means adding polyfills.

The only pure CSS options I could fine required using a wrapper, which isn't possible with sections. I'll have to use JS.

**Pull request merged**